docs/running-calyx/interfacing.md

```diff
@@ -0,0 +1,64 @@
+# Interfacing with Calyx Programs
+
+In order to run RTL designs created from Calyx programs, toplevel `reset` and `go`
+signals must be interfaced with correctly.
+
+Namely:
+1. The `reset` signal must be asserted then deasserted, to initialize the state inside
+control registers correctly.
+2. The `go` signal must be asserted for as long as the module is running. Deasserting
+the `go` signal before a component's `done` signal is asserted will lead to
+[undefined behavior][go-done].
```

Comment:
How about this way to make it a bit more direct, and avoid [passive voice](https://capra.cs.cornell.edu/styleguide/#passive)?

```suggestion
Namely, the client for a Calyx top-level module must:
1. Assert the `reset` signal and then deassert it, to initialize the state inside
control registers correctly.
2. Assert the `go` signal, and keep it asserted as long as the module is running.
3. Wait for the `done` signal to be asserted while keeping `go` high. Deasserting
the `go` signal before a component deasserts its `done` signal will lead to
[undefined behavior][go-done].
```